scripts/create_newcase

```diff
@@ -47,9 +47,15 @@ OR
                         help="Specify a compiler. "
                         "To see list of supported compilers for each machine, use the utility query_config in this directory")
 
+    parser.add_argument("--ncouplers",default=1,
+                        help="Specify number of coupler instances. "
+                        "Set the number of coupler instances in the case. "
+                        "If ncouplers is > 1 then ninst should = 1.")
+
     parser.add_argument("--ninst",default=1,
-                        help="Specify number of component instances"
-                        "Set the number of component instances in the case.")
+                        help="Specify number of component instances. "
+                        "Set the number of component instances in the case. "
+                        "If ninst > 1 then ncouplers should = 1")
```

Comment:
So you should allow the user to either specify --ncouplers OR --ninst and the appropriate ninst or ncouplers from the appropriate input. You should also make it clear from the documentation that if --ncouplers is set to N then the number of instances of each component will also be set to N. The key point is that ninst relates to the number of component instances for a **single coupler.** Not to the total number of component instances. This needs to be spelled out more clearly.
Also having the number of instances by a new argument to create_newcase is a major change in how we address multiple instances. It is assumed that the arguments to create_newcase (compset, grids) cannot be modified once the case is created. This has never been the case with the number of instances which could be modified once the case was created. I assume that you have this as an argument in order to have the correct pe-layout appear when you select multiple instances and that makes sense. Do we want to migrate away from permitting users to change the number of instances after they have created the case. We need to clarify this both as a requirement and in the documentation.